[PrismaCloudV2] Add retries to fetch-incidents API calls (#32433)
demisto/content: Packs/PrismaCloud/ReleaseNotes/4_2_20.md

#### Integrations

##### Prisma Cloud v2

> - Added a retries mechanism to the ***fetch-incidents*** command API calls.

Reviewer: ```suggestion
- Added retries to the API calls in the ***fetch-incidents*** command.
```
